encrypted-media-respec.html

```diff
@@ -620,6 +620,18 @@
       </dl>
     </section>
 
+    <section>
```

Comment:
This appears to be a new top-level section, which seems a bit odd. Also, it is specific to a single method, which is defined in the next top-level section. Should it be part of that? How do other APIs handle this?

Reply:
I wasn't sure where to put it some I'm open to suggestions. I don't think there is any standard place right now but @clelland may have thoughts. I'll move it to the next section.